### PR TITLE
Add AUTHORS.md

### PR DESCRIPTION
**Pull request by Ghabry**
:+1:

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+EasyRPG Player authors
```
Comment (tyrone-sudeium) [inline AUTHORS.md:1]:
```suggestion
EasyRPG Editor authors
```
?

Comment (Ghabry) [reply AUTHORS.md:1]: Copy paste :P
Thanks for reviewing :)

**Review by fmatthew5876 — APPROVED** [review]

**Comment by Ghabry** [comment]
Forgot hellow554. Added now. Created the search Ui and invented the "RpgModel" which my work was based on :)
Now everyone with a "significant" contribution to the editor should be in the list.

**Comment by Ghabry** [comment]
Jenkins: test this please

**Pull request merged**